### PR TITLE
Adding samples and adding reference to gRPC Ecosystem under Community

### PR DESCRIPTION
**Pull request by zinuga**
Adding a set of samples in doc 
Adding reference to gRPC Ecosystem in Community pages

### CODE REVIEW

File: docs/samples/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (hsaliak) [inline docs/samples/index.html]: This file seems to have a mix of markdown and html. How does this render as?
Should this not be a markdown file?

File: community/index.html [hunk 2]
```diff
@@ -10,11 +10,11 @@
     <div class="row">
         <div class="col-md-11 nofloat center-block">
             <div class="community col-lg-9 col-md-10 col-sm-12">
-                <p class="lead">gRPC has an active community of developers who are using, enhancing and building valuable integrations with other software projects. We’d love your help to improve and extend the project. You can reach us via mail, IRC chat, twitter, gitter channel</p>
+                <p class="lead">gRPC has an active community of developers who are using, enhancing and building valuable integrations with other software projects. We’d love your help to improve and extend the project. You can reach us via mail, IRC chat, twitter, gitter channel to start engaging with the project and its members. </p>
```
Comment (hsaliak) [inline community/index.html]: Could you please wrap this line and also add links to twitter and gitter when you mention this here. 
As for IRC we should mention that it's on Freenode,  #grpc

File: docs/samples/index.html (hunk 1 above)
Comment (zinuga) [inline docs/samples/index.html]: Renders fine locally. changing type to markdown

**Comment by hsaliak** [comment]
Ping on this issue

**Comment by zinuga** [comment]
Added links

**Comment by hsaliak** [comment]
What about moving index.html to index.md 
You just have to git mv index.html index.md

**Comment by hsaliak** [comment]
LGTM

**Pull request merged**